pom.xml

```diff
@@ -386,6 +392,25 @@
 					<argLine>${argLine} --enable-preview</argLine>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-scm-plugin</artifactId>
+				<version>2.0.0-M3</version>
+				<configuration>
+					<connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
+					<scmVersionType>tag</scmVersionType>
+					<scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+					<goals>install</goals>
+				</configuration>
+				<executions>
+					<execution>
+						<phase>validate</phase>
```

Comment:
can we configure `package` phase?

Reply:
This phase is not taken into account since the compilation run will not start if maven fails to recognize all dependencies. In this case not having in the m2 directory the common library Maven immediately performs the dependencies check and gives error because rightly it does not find it.